src/components/try-it/call-response/CallResponse.tsx

```diff
@@ -51,3 +55,42 @@ function CopyRequestUrl({ url }: CopyRequestUrlProps) {
         </button>
     );
 }
+
+type CopyResponseButtonProps = {
+    json: string;
+};
+
+function CopyResponseButton({ json }: CopyResponseButtonProps) {
+    const [ copied, setCopied ] = useState(false);
+    const { theme } = useTheme();
+
+    useEffect(() => {
+        const timeout = setTimeout(() => setCopied(false), 2000);
+        return () => clearTimeout(timeout);
+    }, [copied]);
+
+    function copyResponse() {
+        setCopied(true);
+        navigator.clipboard.writeText(JSON.stringify(json, null, 4));
+    }
+
+    return (
+        <button
+            className="absolute rounded-lg right-8 top-20 btn btn-ghost"
+            onClick={copyResponse}
+            type="button"
+        >
+            {copied
+            ? "✔ Copied"
+            : <svg
+                width="24px"
+                height="24px"
+                fill={theme === THEME_DARK ? "#fff" : "#000"}
+                viewBox="0 0 1024 1024"
+            >
+                <path d="M736 768H160V128h576z m-512-64h448V192H224z" />
+                <path d="M864 896H336V736h64v96h400V320h-112v-64h176v640z" />
+            </svg>}
```

Comment:
I'd prefer to see this icon be put in it's own component, inside the `src/components/global/icons` folder using the same types that are in the other icon files.